src/PantheonCalculator.vala

```diff
@@ -45,7 +45,11 @@ namespace PantheonCalculator {
                     window.destroy ();
                 }
             });
+            SimpleAction undo_action = new SimpleAction ("undo", null);
+            undo_action.activate.connect (() => { window.undo (); });
```

Comment:
You can simplify this lambda to:
`undo_action.activate.connect (() => window.undo ());`

Reply:
Did not know about lambda functions, Thanks!